rules/enforce-kubelet-client-tls-authentication-updated/raw.rego

```diff
@@ -0,0 +1,29 @@
+package armo_builtins
+
+# --client-ca-file argument not present in CLI
+deny[msga] {
+	kubelet_info := input[_]
+	kubelet_info.kind == "KubeletInfo"
+	kubelet_info.apiVersion == "hostdata.kubescape.cloud/v1beta0"
+	command := kubelet_info.data.cmdLine
+
+	not contains(command, "--client-ca-file")
+	contains(command, "--config")
```

Comment:
The rule will pass if not `contains(command, "--config")`, is this the correct behavior?

Reply:
no, good catch 👍

Reply:
also, the check has been added